src/pages/BrowseBook/BrowseDocumentModel.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+const {ScriptureParaDocument} = require('proskomma-render');
+
+class BrowseDocumentModel extends ScriptureParaDocument {
+
+    constructor(result, context, config) {
+        super(result, context, config);
+        config.rendered = [];
+        config.blockStack = [];
+        config.currentChapter = null;
+        config.currentVerses = null;
+        addActions(this);
+    }
+}
+
+const addActions = (dInstance) => {
+    dInstance.addAction(
+        'startDocument',
+        () => true,
+        renderer => {
+            renderer.config.nextKey = 0;
+        }
+    );
+    dInstance.addAction(
+        'startBlock',
+        () => true,
+        renderer => {
+            renderer.config.blockStack = [];
+        }
+    );
+    dInstance.addAction(
+        'token',
+        () => true,
+        (renderer, context, data) => {
+            if (data.subType === 'wordLike') {
+                const attScopes = Array.from(context.sequenceStack[0].openScopes)
+                    .filter(s => s.startsWith('attribute'))
+                    .filter(s => ['strong', 'lemma', 'x-strong', 'x-lemma', 'x-content'].includes(s.split('/')[3]))
+                    .map(s => [s.split('/')[3], s.split('/')[5]]);
+                const state = {newSearchString: data.payload};
+                attScopes.forEach(s => state[s[0] = s[1]])
```

Comment:
This line did not make sense, not sure what is intended here. tried printing s, it shows something like this ['strong', 'G2962'] 
So here 'strong' gets replaced by 'G2962', hope that is intended